mkosi

```diff
@@ -1791,6 +1792,63 @@ gpgkey={gpg_key}
 
     reenable_kernel_install(args, root, masked)
 
+@completestep('Installing OpenMandriva')
+def install_openmandriva(args: CommandLineArguments, root: str, do_run_build_script: bool) -> None:
+    release = args.release.strip("'")
+    masked = disable_kernel_install(args, root)
+
+    # OpenMandriva does not (yet) have RPM GPG key on the web
+    gpg_key = '/etc/pki/rpm-gpg/RPM-GPG-KEY-OpenMandriva'
+    if os.path.exists(gpg_key):
+        gpg_key = f'file://{gpg_key}'
+    else:
+        die(f'Missing RPM GPG key, please install distro-release-repo-keys.')
```

Comment:
This is actually not a great idea since it immediately exits the mkosi process without doing any cleanup at all. Maybe replace it with a gpgcheck=0 in the dnf config? That way we can run openmandriva on the CI even without access to the gpg key.

Reply:
Fixed